runtime/testy/studentAPI.py

```diff
@@ -75,7 +80,7 @@ def get_value(self, name, gamepad_number=0):
 
 class Robot(StudentAPI):
   deviceType_to_validParams = {
-    10 : ["duty_cycle", "enable"], 
+    10 : ["duty_cycle", "enable", "enc_pos", "enc_vel", "motor_current"],
```

Comment:
My pull request #193 should include all the relevant fields that as far as I know should be valid params. It should also auto-subscribe to all the fields we want to.